### PR TITLE
Add new known issue in 2.5

### PR DESCRIPTION
**Pull request by oafischer**
https://github.com/stolostron/backlog/issues/22668#issue-1242162745

### CODE REVIEW

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -655,6 +655,17 @@ oc get managedclusterinfo <clusterName> -n <clusterNamespace> -o json | jq -r '.
 
 For more information about the `LoadBalancer` types, see the _Service_ page in the link:https://kubernetes.io/docs/concepts/services-networking/service[Kubernetes documentation.]
 
+[#search-placement-not-working]
+=== Cannot use Search placement in Advanced Configuration Table
+// 2.5:22668
+
+You cannot use the *Search placement* button in the Advanced Configruation Table because the button contains an invalid API version.
```
Comment (swopebe) [inline release_notes/known_issues.adoc]: Making the following changes:

_Advanced Configuration_ table (See the highlighting section of the manuals for recommended ways to highlight a tab or console component)

We rarely highlight a button, more we highlight the function. The button would not contain the invalid API, but the feature or component would. The bold emphasis on a clickable item is correct. See how I reworded.

File: release_notes/known_issues.adoc [hunk 2]
```diff
@@ -655,6 +655,17 @@ oc get managedclusterinfo <clusterName> -n <clusterNamespace> -o json | jq -r '.
...
+=== Cannot use Search placement in Advanced Configuration table
+// 2.5:22668
+
+You cannot use _Search placement_ in the _Advanced Configuration_ table because the feature contains an invalid API version.
+
+To work around this issue, complete the following steps:
+
+. Remove the API version from the search url
+. Delete the preset filters and search manually
+
```
Comment (swopebe) [inline release_notes/known_issues.adoc:668]: need punctuation here

Comment (oafischer) [reply release_notes/known_issues.adoc:668]: Added punctuation and noted how you reworded the section above. Thanks for editing!

**Review by swopebe — APPROVED** [review]

**Comment by swopebe** [comment]
If that is a button, then you are correct to bold it.

**Comment by swopebe** [comment]
Merge when ready

**Pull request merged**